src/app/shared/components/Field/PlayerCard/index.tsx

```diff
@@ -26,31 +27,29 @@ const PlayerCard = (props: PlayerCardProps) => {
       setSelectedTeamSheetLayoutId,
     },
   } = teamContextValue
+
+  const playerSelected = !!player?.id
```

Comment:
can we rename both this var and the prop on line 47 to `isPlayerSelected`?

Reply:
yep like it